docs/pages/release_notes.rst

```diff
@@ -2,6 +2,17 @@
 Release notes
 #############
 
+***************
+Peanut (v6.1.7)
+***************
```

Comment:
I usually also add the v5.4.x release notes here, but it's more important that they get added to the master branch